roles/deploy_code/tasks/cleanup.yml

```diff
@@ -96,6 +86,32 @@
     - deploy_code.mount_type == "squashfs"
   run_once: true
 
+- name: Check if we have a SquashFS image already there.
```

Comment:
I think this needs a `when` clause, because in `_init` we have this:
```yml
- name: Define image builds base path.
  ansible.builtin.set_fact:
    build_base_path: "/home/{{ deploy_user }}/builds/{{ project_name }}_{{ build_type }}"
  when:
    - deploy_code.mount_type is defined
    - deploy_code.mount_type == "squashfs"
```
So `build_base_path` will only be set when squashfs is being used.

And I think we had to do that in `_init` as it was causing syncs to fail.